kite-data/kite-data-core/src/test/java/org/kitesdk/data/event/thrift/Value.java

```diff
@@ -0,0 +1,427 @@
+/**
+ * Autogenerated by Thrift Compiler (0.7.0)
```

Comment:
The definition is checked in and I added a profile to regenerate this to the pom. The reason why I checked this in is to avoid making the build depend on having the thrift compiler installed, which would needlessly complicate our builds when all we really need is a single test file that won't change much.